wagtail/contrib/search_promotions/forms.py

```diff
@@ -45,12 +67,36 @@ class Meta:
 
 
 class SearchPromotionsFormSet(SearchPromotionsFormSetBase):
-    minimum_forms = 1
-    minimum_forms_message = _(
+    """
+    Form set for search promotions
+
+    Attributes
+
+        - minimum_forms (int): Number of forms required for this set.
+    """
+
+    # declare the constants here so we can use them in tests
+    # if the text changes, we don't need to update the tests
```

Comment:
```suggestion
```

This can be removed, it add not useful context.
Additionally, a comment that says 'keep for tests' is not useful for anyone.

In fact, I would say it's better to duplicate the text in the tests, this is the pattern used throughout the entire project and it creates a cross-check for changes such as if things are removed.

Finally, please reset these to the current lower_snake_case convention, there is no need for these to change to class attributes, best to keep this code as is.